script/upgrades/UpgradeYieldDistributor.s.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.8.20;
+
+import {Upgrades} from "openzeppelin-foundry-upgrades/Upgrades.sol";
+import "forge-std/Script.sol";
+import {Options} from "openzeppelin-foundry-upgrades/Options.sol";
+
+contract DeployYieldDistributor is Script {
+    function run() external {
+        vm.startBroadcast();
+        address proxyAddress = 0xeE95A62b749d8a2520E0128D9b3aCa241269024b;
```

Comment:
might be good to make this an argument for the script

Reply:
addressed with ec8e555